### ISSUE TITLE
random failing org.eclipse.core.tests.runtime.jobs.IJobManagerTest.testJobFamilyJoinRepeating

### ISSUE THREAD

**Issue by jukzi** [eclipse-platform/eclipse.platform#31]
https://download.eclipse.org/eclipse/downloads/drops4/I20220506-0640/testresults/html/org.eclipse.core.tests.runtime_ep424I-unit-win32-java11_win32.win32.x86_64_11.html#org.eclipse.core.tests.runtime

```
1.2 expected:<25> but was:<1>junit.framework.AssertionFailedError: 1.2 expected:<25> but was:<1>
at junit.framework.Assert.fail(Assert.java:57)at junit.framework.Assert.failNotEquals(Assert.java:329)
at junit.framework.Assert.assertEquals(Assert.java:78)at junit.framework.Assert.assertEquals(Assert.java:234)
at junit.framework.TestCase.assertEquals(TestCase.java:377)
at org.eclipse.core.tests.runtime.jobs.IJobManagerTest.testJobFamilyJoinRepeating(IJobManagerTest.java:1087)
```

**Comment by fedejeanne** [comment]
Is this Issue still reproducible or was it fixed by #397 ?

**Comment by jukzi** [comment]
let's assert it is solved. if not just reopen this ticket.
